ev3dev2/display.py

```diff
@@ -195,8 +195,17 @@ class Display(FbMem):
     def __init__(self, desc='Display'):
         FbMem.__init__(self)
 
+        if self.var_info.bits_per_pixel == 1:
+            im_type = "1"
+        elif self.var_info.bits_per_pixel == 16:
+            im_type = "RGB"
+        elif self.var_info.bits_per_pixel == 32:
```

Comment:
Hmm... we can probably get away with this since there aren't currently any other XRGB displays on ev3dev devices, but if there ever are, then they won't be usable in color.

Reply:
Do you suggest to use platform here instead of `bits_per_pixel`?

Reply:
I suggest using both.

```
elif self.var_info.bits_per_pixel == 32:
    if platform == EV3:
        grayscale
    else:
        really 32-bit
```

Reply:
69accdf does something similar. Tested on the EV3.